financial-connections/src/main/java/com/stripe/android/financialconnections/navigation/Destination.kt

```diff
@@ -43,12 +44,10 @@ import com.stripe.android.financialconnections.presentation.parentViewModel
  * Represents a destination in the financial connections flow.
  *
  * @param route The route of the destination.
- * @param closeWithoutConfirmation Whether the destination should be close without showing a confirmation modal.
  * @param logPaneLaunched Whether the destination is a real pane that should log the pane launched event.
  */
 internal sealed class Destination(
     protected val route: String,
-    val closeWithoutConfirmation: Boolean,
```

Comment:
This is now part of `TopAppBarState`, since the institution picker can now close without confirmation in _some_ cases, but not all the time.